### ISSUE TITLE
New Feature Suggestions

### ISSUE THREAD

**Issue by fengfeifang** [polywock/globalSpeed#481]
I don't speak good English, so I use a translator and hope I can express myself clearly.

There are actually two types of acceleration, one is to increase a fixed multiplier and the other is to double on the multiplier.
An example:
For example, the default playback speed is 1. The current way to accelerate is to press accelerate and the multiplier becomes 2, then press again to become 3, then press again to become 4.
There is another way to speed up, the default playback speed is 1, press to double it to 2, press again to double it to 4, press again to double it to 8.

I need both types of acceleration, so can you add a second type of acceleration to make it more versatile?

Translated with DeepL.com (free version)

**Comment by fengfeifang** [comment]
I would also like to ask, after I adjust the video speed on a website (for example, to 2x speed), when I watch videos on other websites, the speed also becomes 2x. Is there a way to set the speed adjustment to only take effect once?

**Comment by polywock** [comment]
@fengfeifang Yes, you can enable "pin by default" in the options page so that speed changes only affect the current tab. New tabs will be pinned which allows for speed changes to affect only that tab.

**Comment by Joshuanewby671** [comment]
Thanks again for the response

**Comment by LifeJustDLC** [comment]
> I'm currently working on a draft release for Manifest V3 (Chrome Web Store's deadline is June). --polywock #490

Similar: #158 , #451
